src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,6 +111,25 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
+@include breakpoint($small-screen) {
+    .Body.is-responsive-web {
+        .bdl-CollapsibleSidebar-wrapper {
+            position: initial;
+        }
+
+        .bdl-CollapsibleSidebar.is-expanded {
+            width: 100%;
+
+            .bdl-CollapsibleSidebar-nav ul {
+                // On expand, set ul to fullscreen width with no transition for performance.
+                // Use vw because relative parent (custom-scrollbar) is not full screen width
```

Comment:
I find this comment just a little confusing. Could we say something like, "calc a static width because dynamic calculations for width:100% during transitions created performance issues."

Reply:
Yeah, that works for me. Changed it to almost what you said